src/config/navigation.tsx

```diff
@@ -19,7 +19,8 @@ export const withCustomHeader = (): NavigationStackOptions => ({
   headerTitle: '',
   headerStyle: customStyles().headerStyle,
   headerTitleStyle: customStyles().headerTitleStyle,
-  headerLeft: () => {
+  headerLeft: () => null,
```

Comment:
can you try to remove this? I think is't necessary now , if you can't, close conversation